### DISCUSSION TITLE
Turn off or fine-tuning console log

### DISCUSSION THREAD

**Discussion by GitHubDiom** [manoelcampos/cloudsim-plus#286] [Q&A] [answered]
Hey there, 
I just wondering how to turn off the console log like this:
 ```
INFO  200.21: DatacenterSimple: Vm 0 destroyed on Host 0/DC 1. 
INFO  Simulation: No more future events
```

------

it solved, sorry.

**Comment by manoelcampos** [comment, marked as answer]
You can customize logging for different entities. Check [this issue](https://github.com/manoelcampos/cloudsim-plus/issues/233), there is a complete example available.
